src/farbfeld.rs

```diff
@@ -101,6 +91,76 @@ impl<R: Read> Read for FarbfeldReader<R> {
     }
 }
 
+impl<R: Read + Seek> Seek for FarbfeldReader<R> {
+    fn seek(&mut self, pos: SeekFrom) -> io::Result<u64> {
+        fn checked_seek<F: FnOnce(u64) -> io::Result<()>>(from: u64, by: i64, cbk: F) -> io::Result<bool> {
```

Comment:
I don't see the need for a callback closure here, this could just as well return `off` and let the caller handle what to do with the `u64` offset.

Reply:
Done